src/screens/TopicPageScreen/index.js

```diff
@@ -426,7 +426,9 @@ const TopicPageScreen = (props) => {
 
       <View
         style={{
-          marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0
+          marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0,
+          minHeight: 548,
+          height: 700
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `minHeight` and `height` properties are set to fixed values. This could lead to issues with different screen sizes or orientations. Consider using dynamic sizing based on the device's dimensions or a relative size unit like percentage.
```diff
-          minHeight: 548,
-          height: 700
+          minHeight: Dimensions.get('window').height * 0.7, // 70% of screen height
+          height: Dimensions.get('window').height * 0.9 // 90% of screen height
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->